django/forms/jinja2/django/forms/field.html

```diff
@@ -1,10 +1,10 @@
 {% if field.use_fieldset %}
-  <fieldset>
+  <fieldset{% if field.help_text and field.auto_id and "aria-describedby" not in field.field.widget.attrs %} aria-describedby="{{ field.auto_id }}_helptext"{% endif %}>
   {% if field.label %}{{ field.legend_tag() }}{% endif %}
 {% else %}
   {% if field.label %}{{ field.label_tag() }}{% endif %}
 {% endif %}
-{% if field.help_text %}<div class="helptext"{% if field.id_for_label %} id="{{ field.id_for_label}}_helptext"{% endif %}>{{ field.help_text|safe }}</div>{% endif %}
+{% if field.help_text %}<div class="helptext"{% if field.auto_id %} id="{{ field.auto_id }}_helptext"{% endif %}>{{ field.help_text|safe }}</div>{% endif %}
```

Comment:
Why did we globally change `id_for_label` to `auto_id`? :thinking: It looks like an unrelated cleanup (or even if it's related, it's something that can probably be processed separately).

Reply:
Sure. I'll pull this out into a separate PR and explain the reason for this change.

Reply:
Refs https://github.com/django/django/pull/17480

Reply:
Thanks, please rebase.